core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -2605,7 +2606,13 @@ private void checkPredicates(RelOptCluster cluster, RelOptTable empTable,
     final String sql = "select cast(null as integer) as a\n"
         + "union all\n"
         + "select 5 as a";
-    final Union rel = (Union) sql(sql).toRel();
+    final Union rel = (Union) sql(sql)
+        .withFactory(a ->
+            a.withSqlToRelConfig(
```

Comment:
can you factor out this configuration into a single function to avoid repeating this pattern?